circle.yml

```diff
@@ -38,10 +38,12 @@ test:
     # TESTS:
     # first, run ctest inside a container
     - docker run -p 1204:1204 hyperledger/iroha-docker /test.sh
+
+    - docker run -d -p 1204:1204 hyperledger/iroha-docker /run.sh && sleep 5 && curl http://127.0.0.1:1204
```

Comment:
Ok thanks, I will  remove.
And currently, this test is failed, I'm trying to fix it. 🏃 
```
curl -X POST http://127.0.0.1:1204/account/register -d '{"publicKey":"WdvM/DPabapmtA7ISbTYPywbHxk8gWu2221LzmcmAgw=","alias":"yonezu","timestamp":1482053586}' 
```